wp-content/themes/workingnyc/package.json

```diff
@@ -17,6 +17,7 @@
     "axios": "^0.25.0",
     "regenerator-runtime": "^0.13.9",
     "vue": "^2.6.12",
+    "vue-carousel": "^0.18.0",
```

Comment:
It doesn't look like you're using vue-carousel, you can remove this

Reply:
done